test/jdk/java/nio/channels/Channels/TransferTo2.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The copyright years need to be updated per the usual convention in all files in this PR.

Reply:
Thanks a lot for this hint! I will do that at thanks. Apparently this happened because the PR was created in 2021 actually.

Reply:
I have pushed commit https://github.com/openjdk/jdk/pull/6711/commits/4bd0d545414756276dff6d2d4f43167cc451425f which updates the copyright year of all touched files.

Reply:
Looks correct now.